src/Site/OAuth/OAuthBase.php

```diff
@@ -37,7 +38,10 @@ abstract protected function getOAuthProvider($redirectUri): AbstractProvider;
     public function oauthCallback(Request $request, Application $app)
     {
         $website = Website::get();
-        $redirectUri = $website->baseUrl() . '/oauthcallback/'. $this->getProviderName();
+        // TODO: Can we assume https here and get rid of the Website::get() call entirely? Or are there times when we need http here?
```

Comment:
yes, we can assume HTTPS.  Safe to delete this comment.

Reply:
Done in latest commit.